site/content/docs/5.2/components/close-button.md

```diff
@@ -22,13 +22,19 @@ Disabled close buttons change their `opacity`. We've also applied `pointer-event
 <button type="button" class="btn-close" disabled aria-label="Close"></button>
 {{< /example >}}
 
-## White variant
+## Dark mode
 
-Change the default `.btn-close` to be white with the `.btn-close-white` class. This class uses the `filter` property to invert the `background-image`.
+{{< callout info >}}
+**Heads up!** As of v5.3.0, the `.btn-close-white` class is deprecated. Instead, use `data-bs-theme="dark"` to change the color mode of the close button.
+{{< /callout >}}
+
+Add `data-bs-theme="dark"` to the `.btn-close`, or to it's parent element, to invert the close button. This uses the `filter` property to invert the `background-image` without overriding its value.
 
 {{< example class="bg-dark" >}}
-<button type="button" class="btn-close btn-close-white" aria-label="Close"></button>
-<button type="button" class="btn-close btn-close-white" disabled aria-label="Close"></button>
+<div data-bs-theme="dark">
```

Comment:
This example doesn't work when the page is in dark mode. If we'd like to keep this part of the documentation we should display `data-bs-theme="light"` when the page is in dark mode.
Maybe just do in this part the same thing as in the carousel and just explain that `btn-close-white` is deprecated.

Reply:
We'd need to add some custom styles to enable that kind of behavior. As it stands now, this wouldn't work.